lib/active_model/serializer/field.rb

```diff
@@ -0,0 +1,56 @@
+module ActiveModel
+  class Serializer
+    # Holds all the meta-data about a field (i.e. attribute or association) as it was
```

Comment:
Agreed that it's not super important, but as I like grammar as well: a field is *defined* as "either an attribute or an association".